iml-services/iml-snapshot/src/main.rs

```diff
@@ -3,15 +3,126 @@
 // license that can be found in the LICENSE file.
 
 use futures_util::stream::TryStreamExt;
+use iml_command_utils::{wait_for_cmds_success, CmdUtilError};
+use iml_graphql_queries::snapshot as snapshot_queries;
+use iml_manager_client::{get_influx, graphql, ImlManagerClientError};
```

Comment:
```suggestion
use iml_manager_client::{get_influx, graphql, Client, ImlManagerClientError};
```

This allows us to remove the `reqwest` dep from this crate.

Reply:
Done